### PR TITLE
Migration of Community Guides

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

The community guides, tutorials and/or ecosystem related documentation is being moved out of our documentation and will get a new home.

This PR is in draft; in preparation of this migration, and for me to keep track of migrated parts.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration/remote.markdown [hunk 1]
```diff
@@ -32,5 +32,3 @@ Just putting a port up is not secure. You should definitely consider encrypting
 </div>
 
 Protect your communication with a [self-signed certificate](/docs/ecosystem/certificates/tls_self_signed_certificate/) between your client and the Home Assistant instance.
-
-For another way to access your Home Assistant frontend, check out [the instructions how to use Tor](/docs/ecosystem/tor/).
```
Comment (fabaff) [inline source/_docs/configuration/remote.markdown:36]: The link should point to the new location.

File: source/_docs/configuration/securing.markdown [hunk 2]
```diff
@@ -15,7 +15,7 @@ Here's the summary of what you *must* do to secure your Home Assistant system:
 
 ## Remote Access
 
-If you want secure remote access, the easiest option is to use [Home Assistant cloud](/cloud/). Other options are to use [TLS/SSL](/docs/ecosystem/certificates/lets_encrypt/) to expose your instance to the internet, use a [VPN](https://pivpn.dev/), [Tor](/docs/ecosystem/tor/) or an [SSH tunnel](/blog/2017/11/02/secure-shell-tunnel/).
+If you want secure remote access, the easiest option is to use [Home Assistant cloud](/cloud/). Other options are to use [TLS/SSL](/docs/ecosystem/certificates/lets_encrypt/) to expose your instance to the internet, use a [VPN](https://pivpn.dev/) or an [SSH tunnel](/blog/2017/11/02/secure-shell-tunnel/).
```
Comment (fabaff) [inline source/_docs/configuration/securing.markdown:22]: Dito

File: source/_docs/installation.markdown [hunk 3]
```diff
@@ -106,10 +106,4 @@ These guides are provided as-is. Some of these install methods are more limited
     </div>
     <div class='title'>Synology</div>
   </a>
-  <a class='option-card' href='/docs/installation/freenas/'>
-    <div class='img-container'>
-      <img src='/images/supported_brands/freenas.png' />
-    </div>
-    <div class='title'>FreeNAS</div>
-  </a>
```
Comment (fabaff) [inline source/_docs/installation.markdown:114]: If you want to make an installation of Home Assistant then you will look in the documentation and not the forum. 

The section which contains that guide was renamed to "Community provided guides" at some point but actually they are OS-specify guide. That they are not "official" but definitely part of "Home Assistant provides multiple ways to be installed." and we don't want you to re-invent the wheel again especially as using a `venv` is mentioned as a way to run HA.
.

**Review by fabaff — CHANGES_REQUESTED** [review]

File: source/_docs/installation.markdown (hunk 3 above)
Comment (frenck) [reply source/_docs/installation.markdown:114]: It is being removed.

File: source/_docs/configuration/remote.markdown (hunk 1 above)
Comment (frenck) [reply source/_docs/configuration/remote.markdown:36]: I think having DuckDNS in here is what we document on multiple places. Tor not so much. Pointing to a community guide (which it has been moved to) is something we should not do. At least, not to a specific guide.

Lets see how the community guide evolve and maybe later link it up.

File: source/_docs/configuration/securing.markdown (hunk 2 above)
Comment (frenck) [reply source/_docs/configuration/securing.markdown:22]: Dito

**Review by balloob — APPROVED** [review]

**Pull request merged**